tests/test_cases/test_compare/test_compare.py

```diff
@@ -11,7 +11,7 @@
 from cocotb.triggers import RisingEdge, FallingEdge
 
 
-class Testbench(object):
+class Testbench():
```

Comment:
This change is good, but I'd be inclined to drop the `()` everywhere too.

I also think that #2423 probably does this for us anyway.